nixos/modules/services/games/minecraft-server.nix

```diff
@@ -180,6 +206,9 @@ in {
         Restart = "always";
         User = "minecraft";
         WorkingDirectory = cfg.dataDir;
+      } // optionalAttrs (cfg.fifo != null) {
+        StandardInput = "socket";
```

Comment:
From what I know about systemd sockets, this will only work when the service has been activated by the socket itself? I believe with this setting, doing `systemctl restart minecraft-server` should fail, since it wasn't activated by the socket.

Reply:
This is only true (as far as I know) for services with `@` at the end of the name. I tested it, and I can restart the server with `systemctl restart minecraft-server`.

The socket unit is just there to create the FIFO device.